src/helpers/transport/query.rs

```diff
@@ -300,7 +300,7 @@ impl Default for IpaQueryConfig {
     fn default() -> Self {
         Self {
             per_user_credit_cap: 3,
-            max_breakdown_key: 64,
+            max_breakdown_key: 20,
```

Comment:
We only accept max_breakdown_key <= 32 now. Using 20 here which was the value used in helper_network integration tests.

Reply:
may be good to create a type for it to enforce and document why

Reply:
I chatted with Ben Savage and we think this <=32 limitation is temporary. We have the assert to check the value is within the limit, so it should be okay with u32.